app/views/layouts/provider.html.slim

```diff
@@ -16,7 +16,7 @@ html[lang="en"]
   body
     #wrapper.pf-c-page
       = render partial: '/shared/provider/header'
-      = render partial: 'shared/provider/navigation/vertical_nav' unless active_menu == :dashboard
+      = render partial: 'shared/provider/navigation/vertical_nav', :locals => {service: @service} unless active_menu == :dashboard
```

Comment:
In some views we may not have the `@service`.

Reply:
Let's fix in:
https://github.com/3scale/porta/blob/faeb8446346471aae12f4aa5502e36a4605d8c56/app/helpers/application_helper.rb#L369-L371

Make it:
```ruby
def has_out_of_date_configuration?(service)
  return unless service
  service.pending_affecting_changes?
end
```